src/workflows-app/components/FilterableWorkflowTable.ts

```diff
@@ -151,6 +191,64 @@ const FilterableWorkflowTable = ({
     }
   };
 
+  const includeKey = useMemo(
+    () => [
+      'backendStatus',
+      'executionStatus',
+      'shardIndex',
+      'jobId',
+      'start',
+      'end',
+      'stderr',
+      'stdout',
+      'tes_stdout',
+      'tes_stderr',
+      'attempt',
+      'subWorkflowId', // needed for task type column
+    ],
+    []
+  );
+  const excludeKey = useMemo(() => [], []);
+
+  const getKeys = (keyValuePairs: {}): undefined => {
+    for (const key of Object.getOwnPropertyNames(keyValuePairs)) {
+      keyValuePairs[key];
+      delete Object.assign(keyValuePairs, {
+        [parseMethodString(key).variable]: keyValuePairs[key],
+      })[key];
+    }
+  };
+
+  const loadWorkflows = useCallback(
+    async (workflowId: string): Promise<WorkflowMetadata | void> => {
+      try {
+        const { cromwellProxyUrlState } = await loadAppUrls(workspaceId, 'cromwellProxyUrlState');
+        if (cromwellProxyUrlState.status === AppProxyUrlStatus.Ready) {
+          return await fetchMetadata(cromwellProxyUrlState.state, workflowId, signal, includeKey, excludeKey);
+        }
+      } catch (error) {
+        notify('error', 'Error loading run details', {
+          detail: error instanceof Response ? await error.text() : error,
+        });
```

Comment:
Suggest using `reportError` from `libs/error`, which will do this for you.

```suggestion
        reportError('Error loading run details', error);
```